nilearn/datasets/neurovault.py

```diff
@@ -0,0 +1,2589 @@
+"""
+Download statistical maps available on Neurovault (http://neurovault.org).
+"""
+# Author: Jerome Dockes
+# License: simplified BSD
+
+import os
+import logging
+import warnings
+from copy import copy, deepcopy
+import shutil
+import re
+import json
+from glob import glob
+from tempfile import mkdtemp
+from collections import Container
+try:
+    from collections import OrderedDict
```

Comment:
Is the try/except because of Python 2.6 support?

We should really drop Python 2.6 support.